cluster-scope/overlays/prod/osc/osc-cl2/kustomization.yaml

```diff
@@ -13,13 +13,16 @@ resources:
   - ../../../../base/core/namespaces/odh-trino
   - ../../../../base/core/namespaces/openshift-nfd
   - ../../../../base/core/namespaces/nvidia-gpu-operator
+  - ../../../../base/core/namespaces/reloader
   - ../../../../base/operators.coreos.com/operatorgroups/openshift-nfd
   - ../../../../base/operators.coreos.com/subscriptions/nfd
   - ../../../../base/operators.coreos.com/subscriptions/opendatahub-operator
   - ../../../../base/config.openshift.io/oauths/cluster
   - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/acme-operator
   - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/cluster-admins-rb
+  - ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/reloader-reloader-role-binding
   - ../../../../base/rbac.authorization.k8s.io/clusterroles/acme-operator
+  - ../../../../base/rbac.authorization.k8s.io/clusterroles/reloader-reloader-role
```

Comment:
include the bundle instead: https://github.com/operate-first/apps/tree/master/cluster-scope/bundles/reloader

example: 
https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml#L130